dhis2-android-app/src/main/java/org/dhis2/mobile/processors/ReportUploadProcessor.java

```diff
@@ -87,10 +88,22 @@ public static void upload(Context context, DatasetInfoHolder info, ArrayList<Gro
                         context.getString(R.string.import_failed));
             }
 
+            message = message + description;
```

Comment:
@idelcano this code for log mixed with other logic is a little dirty, can you extract a separate method or class?